### ISSUE TITLE
where is dataset mask?

### ISSUE THREAD

**Issue by shoutOutYangJie** [ZhenbangDu/Reliable_AD#2]
I download dataset and I find that "png" data has no alpha channel.

**Comment by ZhenbangDu** [comment]
Hi. The PNG images in the test set already include mask channels. For the training set, the corresponding masks are embedded in separate product images with transparent backgrounds. These are currently being prepared and will be uploaded soon.

**Comment by ZhenbangDu** [comment]
Hi, we’ve released a subset of .png product images with an alpha channel (~362k images). You can find them at this link: https://3.cn/10hI-gq6q.
